src/codecs/avif/encoder.rs

```diff
@@ -101,20 +99,18 @@ impl<W: Write> AvifEncoder<W> {
         color: ColorType,
     ) -> ImageResult<()> {
         self.set_color(color);
-        let config = self.config;
+        let encoder = self.encoder.clone();
```

Comment:
I think this clone could be avoided

Reply:
It can. I put the clone there because I thought there was a good reason for the mutable borrow the compiler complained about, but looking closer I don't think there is.